plugins/usertiming.js

```diff
@@ -0,0 +1,93 @@
+/**
+\file usertiming.js
+Plugin to collect metrics from the W3C User Timing API.
+For more information about User Timing,
+see: http://www.w3.org/TR/user-timing/
+*/
+
+(function() {
+
+	BOOMR = BOOMR || {};
+	BOOMR.plugins = BOOMR.plugins || {};
+
+	/**
+	 * @returns String compressed user timing data that occurred since the last call
+	 */
+	function getUserTiming() {
+		var now = BOOMR.now();
+
+		map = UserTimingCompression.getCompressedUserTiming(impl.options);
+		res = UserTimingCompression.compressForUri(map);
+		impl.options["from"] = now;
+
+		return res;
+	}
+
+	var impl = {
+		complete: false,
+		initialized: false,
+		supported: false,
+		options: {"from": 0},
+
+		done: function() {
+			var r;
+
+			if (this.complete) {
+				return;
+			}
+
+			BOOMR.removeVar("usertiming");
+			r = getUserTiming();
+			if (r) {
+				BOOMR.info("Client supports User Timing API", "usertiming");
+				BOOMR.addVar({
+					"usertiming": JSON.stringify(r)
```

Comment:
if `r` is already passed through `compressForUri`, then I don't think we need `JSON.stringify` here.